api/app/services/ingestions/strategies/epub.rb

```diff
@@ -0,0 +1,100 @@
+module Ingestions
+  module Strategies
+    class Epub < Ingestions::Strategies::AbstractStrategy
+
+      def perform
+        compose Fetchers::URL, extension: "epub" if epub_url?
```

Comment:
This is wrong. Anything involved with fetching the source file should live outside a strategy. The ingestion source directory should be fully in place before the strategy begins it's work. See https://manifold-community.slack.com/files/U8KBJCT8B/FB2BCB08G/image.png